x/intertx/types/v1/features/msg_submit_tx.feature

```diff
@@ -0,0 +1,52 @@
+Feature: MsgSubmitTx
+
+  Scenario: a valid message
+    Given the message
+    """
+    {
+      "owner": "regen1nzh226hxrsvf4k69sa8v0nfuzx5vgwkczk8j68",
+      "connection_id": "channel-5"
+    }
+    """
+    And a valid tx for msg
```

Comment:
Can we include message bytes in the example? This is a bit unclear to me from a documentation perspective.

Reply:
sure https://github.com/regen-network/regen-ledger/pull/1477/commits/3cdb045248e61ca9b3cd716806e08453a8508eb2